app-emulation/box64/autobuild/defines

```diff
@@ -0,0 +1,31 @@
+PKGNAME=box64
+PKGSEC=utils
+PKGDES="an userspace x86_64 emulator with a twist, aimed to run x86_64 Linux programs on non-x86_64 Linux systems"
+PKGDEP="glibc gcc-runtime"
+ABTYPE="cmakeninja"
+BUILDDEP="mold"
+
+FAIL_ARCH="(amd64|loongson3)"
+
+# No need to regenerate wrapper,-DCI=ON
+CMAKE_AFTER__PPC64EL=(
+    "-DPPC64LE=ON"
+    "-DCI=ON"
+    "-DWITH_MOLD=ON"
+)
+CMAKE_AFTER__LOONGARCH64=("-DLARCH64=ON"
+    "-DLARCH64_DYNAREC=ON"
+    "-DCI=ON"
+    "-DWITH_MOLD=ON"
+)
+CMAKE_AFTER__RISCV64=(
+    "-DRV64=ON"
+    "-DRV64_DYNAREC=ON"
+    "-DCI=ON"
+    "-DWITH_MOLD=ON"
+)
+CMAKE_AFTER__ARM64=("-DARM64=ON"
```

Comment:
Can you at least check your code styling?

Also, `CMAKE_AFTER__*` is meant as an overlay:

```
CMAKE_AFTER=(
    '-DCI=ON'
    '-DWITH_MOLD=ON'
)
CMAKE_AFTER__ARM64=(
    "${CMAKE_AFTER[@]}"
    '-DARM64=ON'
    '-DARM_DYNAREC=ON'
)
...
```

Also, at least sort the variables by name so it's easier to find?